server/main.js

```diff
@@ -123,40 +125,43 @@ Meteor.publish('tabular_getInfo', function (tableName, selector, sort, skip, lim
     const paths = getSearchPaths(table);
     const newSort = transformSortArray(findOptions.sort);
 
-    filteredRecordIds = table.searchCustom(
+    filteredRecordIdsPromise = Promise.resolve( table.searchCustom(
       this.userId,
       newSelector,
       tokens,
       paths,
       newSort,
       skip,
       limit
-    );
+    ));
   } else {
     filteredCursor = table.collection.find(newSelector, findOptions);
```

Comment:
Doesn't `find` require findAsync ? I've not looked much at meteor 3.0 but thought all the interface to a collection now had async versions ?

Reply:
Right, apologies! I've not tried to migrate anything myself yet.